maestro/tests/yamls/agents/dry_run_paralle_agent.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: maestro/v1alpha1
+kind: Agent
+metadata:
+  name: test1
+  labels:
+    app: test-example
```

Comment:
Aren't you missing `framework: bee`?

Reply:
And same for other agents?

Reply:
I'll add it.  The dry run doesn't check the `framework` but it should be there.  Thanks!